packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -69,17 +71,29 @@ export function signInActor({ services }: SignInMachineOptions) {
                 onDone: [
                   {
                     cond: 'shouldSetupTOTP',
-                    actions: ['setUser', 'setChallengeName'],
+                    actions: [
+                      'setUser',
+                      'setChallengeName',
+                      'setRequiredAttributes',
```

Comment:
For my understanding -- can there be `setRequiredAttributes` for `setupTOTP` and and `shouldConfirmSignIn`. If not, should we only call that action for `forceChangePassword`?

Reply:
Yeah, right now we assume it only works for `forceChangePassword` I can delete those.  Maybe in the future we'll find a scenario we'll need it for setupTOTP and shouldConfirmSignIn, but not right now